deeppavlov/models/go_bot/launch_train_bot.py

```diff
@@ -0,0 +1,163 @@
+import json
+import os
+import argparse
+import logging
+
+from logging import getLogger
+
+from deeppavlov.core.common.file import read_json
+from deeppavlov import configs, evaluate_model, train_model
+from deeppavlov.dataset_readers.dstc2_reader import SimpleDSTC2DatasetReader
+from deeppavlov.download import download_decompress
+from deeppavlov.core.data.sqlite_database import Sqlite3Database
+from deeppavlov.dataset_iterators.dialog_iterator import DialogDatasetIterator
+
+
+logging.basicConfig(level=logging.INFO,
+                    format="[%(levelname)s] [%(asctime)s] %(message)s",
+                    datefmt="%y-%m-%d %H:%M:%S")
+
+log = getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+formatter = logging.Formatter('[%(levelname)s] [%(asctime)s] %(message)s',"%y-%m-%d %H:%M:%S", "%")
+fh = logging.FileHandler("log")
+fh.setFormatter(formatter)
+log.addHandler(fh)
+
+
+def load_database(bot_data_dir, data):
+    database = Sqlite3Database(
+        primary_keys=["name"],
+        save_path=os.path.join(bot_data_dir, "db.sqlite")
+    )
+
+    iterator = DialogDatasetIterator(data)
+    db_results = []
+
+    for dialog in iterator.gen_batches(batch_size=1, data_type='all'):
+        turns_x, turns_y = dialog
+        db_results.extend(x['db_result'] for x in turns_x[0] if x.get('db_result'))
+
+    print(f"Adding {len(db_results)} items.")
+    if db_results:
+        database.fit(db_results)
+
+
+def load_bot_data(bot_data_dir, max_train_data):
+    data = SimpleDSTC2DatasetReader().read(bot_data_dir)
+
+    if max_train_data is not None:
+        src = os.path.join(bot_data_dir, 'simple-dstc2-trn.json')
+        dst = os.path.join(bot_data_dir, 'simple-dstc2-trn_full.json')
+
+        with open(src, 'rt') as fin:
```

Comment:
```suggestion
        with open(src, 'rt', encoding='utf8') as fin:
```
It's better to explicitly set encoding because default value could be different on different systems.